src/test/java/redis/clients/jedis/PipeliningMigrateTest.java

```diff
@@ -0,0 +1,398 @@
+package redis.clients.jedis;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.both;
+import static org.hamcrest.Matchers.containsString;
+import static org.hamcrest.Matchers.equalTo;
+import static org.hamcrest.Matchers.hasItems;
+import static org.hamcrest.Matchers.hasToString;
+import static org.hamcrest.Matchers.instanceOf;
+import static org.junit.Assert.assertArrayEquals;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNull;
+
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+import redis.clients.jedis.commands.jedis.JedisCommandsTestBase;
+import redis.clients.jedis.exceptions.JedisDataException;
+import redis.clients.jedis.params.MigrateParams;
+
+public class PipeliningMigrateTest extends JedisCommandsTestBase {
```

Comment:
MigratePipeliningTest.

IMO, we can maintain `<command-group>PipeliningTest` pattern.

Reply:
Renamed.